tfhe/src/core_crypto/commons/parameters.rs

```diff
@@ -110,6 +110,14 @@ impl GlweDimension {
     }
 }
 
+impl std::ops::Mul<PolynomialSize> for GlweDimension {
+    type Output = LweDimension;
+
+    fn mul(self, poly_size: PolynomialSize) -> LweDimension {
+        LweDimension(self.0 * poly_size.0)
+    }
+}
+
```

Comment:
Makes it too easy to escape the newtypes, I would prefer

LweDimension::from_glwe_parameters(glwe_dimension, polynomial_size) in that case

Reply:
Or maybe better

On GlweDimension impl

fn to_equivalent_lwe_dimension(self, polynomial_size) -> LweDimension { mul code }